osquery/logger/plugins/kafka_producer.cpp

```diff
@@ -26,17 +23,44 @@
 namespace osquery {
 
 FLAG(string,
-     logger_kafka_brokers,
+     kafka_brokers,
      "localhost",
      "Bootstrap broker(s) as a comma-separated list of host or host:port "
      "(default port 9092)");
+FLAG_ALIAS(std::string, logger_kafka_brokers, kafka_brokers);
 
-FLAG(string, logger_kafka_topic, "", "Kafka topic to publish logs under");
+FLAG(string, kafka_topic, "osquery", "Kafka topic to publish logs under");
+FLAG_ALIAS(std::string, logger_kafka_topic, kafka_topic);
 
 FLAG(string,
-     logger_kafka_acks,
+     kafka_acks,
      "all",
      "The number of acknowledgments the leader has to receive (0, 1, 'all')");
+FLAG_ALIAS(std::string, logger_kafka_acks, kafka_acks);
+
+FLAG(string,
+     kafka_security_protocol,
+     "plaintext",
+     "Protocol used to communicate with brokers ('plaintext', 'ssl')");
+
+FLAG(string, kafka_ssl_cipher_suites, "", "List of cipher suites");
+
+FLAG(string,
+     kafka_ssl_key_location,
+     "",
+     "Path to client's private key (PEM) used for authentication");
+
+FLAG(string, kafka_ssl_key_password, "", "Private key passphrase");
+
+FLAG(string,
+     kafka_ssl_certificate_location,
+     "",
+     "Path to client's public key (PEM) used for authentication");
+
+FLAG(string,
+     kafka_ssl_ca_location,
```

Comment:
Can this use the existing bundle from [`tls_server_certs`](https://github.com/facebook/osquery/blob/master/osquery/remote/transports/tls.cpp#L41)?

Reply:
Yes, I think that should be default value. But I am guessing some of the users are probably using self signed certificates. So might want to override the value. Do you want them to update certs.pem? Or should we keep this option for overriding?

Reply:
My preference is to keep 1 flag that holds either the default `certs.pem` or is set to your own bundle. That bundle will be used for all TLS server verification the osquery client makes.